client/state/data-layer/wpcom/sites/simple-payments/index.js

```diff
@@ -23,7 +23,10 @@ import { metaKeyToSchemaKeyMap, metadataSchema } from 'state/simple-payments/pro
 import { http } from 'state/data-layer/wpcom-http/actions';
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from 'lib/simple-payments/constants';
-import { isValidSimplePaymentsProduct } from 'lib/simple-payments/utils';
+import {
+	isValidSimplePaymentsProduct,
+	decodeProductAttribute,
+} from 'lib/simple-payments/utils';
```

Comment:
lol. good old textarea hack. haven't heard of XSS issues with the hack yet so two thumbs up with using `lib/formating`.